linera-views/src/backends/rocks_db.rs

```diff
@@ -89,89 +90,55 @@ impl RocksDbStoreInternal {
         options.create_if_missing(true);
         let db = DB::open(&options, path)?;
         let root_key = root_key.to_vec();
+        let metrics = Handle::current().metrics();
+        let spawn_blocking = metrics.num_workers() == 1;
```

Comment:
Please let's have `block_in_place: bool` part of `RocksDbStoreConfig` and the config value forwarded all the way here.

Rationale: `block_in_place` may not work for all the use cases (due to `join!`) so we should let the users of `linera-view` decide. The documentation of `RocksDbStore` may suggest setting `block_in_place = (tokio::runtime::Handle::current().metrics().num_workers() > 1)` and should include a pointer to the documentation of  `block_in_place`.